nipype/algorithms/tests/test_confounds.py

```diff
@@ -36,6 +37,21 @@ def test_fd(tmpdir):
         ground_truth, np.loadtxt(res.outputs.out_file, skiprows=1), atol=.16)
     assert np.abs(ground_truth.mean() - res.outputs.fd_average) < 1e-2
 
+    fdisplacement = FramewiseDisplacement(
```

Comment:
Could you make this a new test, named something like `test_fd_riemannian`? Then you can preface it with `@pytest.mark.skipif(...)`, like so:

```Python
# Up near nonitime definition (line 13)
nogeomstats = True
try:
    import geomstats
    nogeomstats = False
except ImportError:
    pass

...

@pytest.mark.skipif(nogeomstats, reason="geomstats is not installed")
def test_fd_riemannian(tmpdir):
    ...
```